Sources/TuistLoader/Models+ManifestMappers/CoreDataModel+ManifestMapper.swift

```diff
@@ -12,7 +12,63 @@ extension TuistCore.CoreDataModel {
     static func from(manifest: ProjectDescription.CoreDataModel, generatorPaths: GeneratorPaths) throws -> TuistCore.CoreDataModel {
         let modelPath = try generatorPaths.resolve(path: manifest.path)
         let versions = FileHandler.shared.glob(modelPath, glob: "*.xcdatamodel")
-        let currentVersion = manifest.currentVersion
+
+        var currentVersion: String!
```

Comment:
I didn't know that the current version is define inside the model. In that case I'd consider deprecating passing the version through the manifest. What about extending [this linter](https://github.com/tuist/tuist/blob/master/Sources/TuistGenerator/Linter/TargetLinter.swift#L25) to print a warning if the value of `currentVersion` is not nil? We can delete it in a few versions.

Reply:
For linting, seeing the model will continue to have a version, the `ManifestLinter` is probably more suited to produce this warning 🙂


https://github.com/tuist/tuist/blob/master/Sources/TuistLoader/Linter/ManifestLinter.swift